pkg/core/localize/locales/en/cmd/kafka.en.toml

```diff
@@ -322,6 +322,9 @@ one = "Cloud Region:"
 [kafka.create.input.plan.message]
 one = 'Instance type:'
 
+[kafka.create.log.info.sizePreview]
+one = 'Selected Kafka size is available under technology preview'
```

Comment:
Not sure about this being good explanation

Reply:
It might be helpful to point to the official tech preview definition:

> This Kafka instance size is a Technology Preview feature. Do not use it for production use. For more information, see https://access.redhat.com/support/offerings/techpreview/.